shared/src/metabase/shared/util/currency.cljc

```diff
@@ -0,0 +1,851 @@
+          :decimal_digits 2,
+          :rounding 0,
+          :code "MXN",
+          :name_plural "Mexican pesos"}],
+   [:MYR {:symbol "RM",
+          :name "Malaysian Ringgit",
+          :symbol_native "RM",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "MYR",
+          :name_plural "Malaysian ringgits"}],
+   [:MZN {:symbol "MTn",
+          :name "Mozambican Metical",
+          :symbol_native "MTn",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "MZN",
+          :name_plural "Mozambican meticals"}],
+   [:NAD {:symbol "N$",
+          :name "Namibian Dollar",
+          :symbol_native "N$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "NAD",
+          :name_plural "Namibian dollars"}],
+   [:NGN {:symbol "₦",
+          :name "Nigerian Naira",
+          :symbol_native "₦",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "NGN",
+          :name_plural "Nigerian nairas"}],
+   [:NIO {:symbol "C$",
+          :name "Nicaraguan Córdoba",
+          :symbol_native "C$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "NIO",
+          :name_plural "Nicaraguan córdobas"}],
+   [:NOK {:symbol "Nkr",
+          :name "Norwegian Krone",
+          :symbol_native "kr",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "NOK",
+          :name_plural "Norwegian kroner"}],
+   [:NPR {:symbol "NPRs",
+          :name "Nepalese Rupee",
+          :symbol_native "नेरू",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "NPR",
+          :name_plural "Nepalese rupees"}],
+   [:NZD {:symbol "NZ$",
+          :name "New Zealand Dollar",
+          :symbol_native "$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "NZD",
+          :name_plural "New Zealand dollars"}],
+   [:OMR {:symbol "OMR",
+          :name "Omani Rial",
+          :symbol_native "ر.ع.‏",
+          :decimal_digits 3,
+          :rounding 0,
+          :code "OMR",
+          :name_plural "Omani rials"}],
+   [:PAB {:symbol "B/.",
+          :name "Panamanian Balboa",
+          :symbol_native "B/.",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "PAB",
+          :name_plural "Panamanian balboas"}],
+   [:PEN {:symbol "S/.",
+          :name "Peruvian Nuevo Sol",
+          :symbol_native "S/.",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "PEN",
+          :name_plural "Peruvian nuevos soles"}],
+   [:PHP {:symbol "₱",
+          :name "Philippine Peso",
+          :symbol_native "₱",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "PHP",
+          :name_plural "Philippine pesos"}],
+   [:PKR {:symbol "PKRs",
+          :name "Pakistani Rupee",
+          :symbol_native "₨",
+          :decimal_digits 0,
+          :rounding 0,
+          :code "PKR",
+          :name_plural "Pakistani rupees"}],
```

Comment:
one day we need to implement counting by indian and east asian customary values (lakhs/crore, wan/man, etc)

probably not this day but one day